### PR TITLE
Headbugfix

### PR DESCRIPTION
**Pull request by ViperMaul**
Adding HeadBugFix button to the ACE Options dialog
#1738

### CODE REVIEW

File: addons/optionsmenu/stringtable.xml [hunk 1]
```diff
@@ -345,6 +345,22 @@
             <German>Protokolliert Debug-Informationen im RPT und speichert sie in der Zwischenablage.</German>
             <Portuguese>Envia informação de depuração para RPT e área de transferência.</Portuguese>
         </Key>
+		<Key ID="STR_ACE_OptionsMenu_headBugFix">
+            <English>Headbug Fix</English>
```
Comment (bux578) [inline addons/optionsmenu/stringtable.xml]: Don't add all languages if the values aren't translated. It's hard to find those instances for translators.

**Comment by bux578** [comment]
Why are spectator commits in here?

**Comment by ViperMaul** [comment]
Something weird that GitHub automatically. As discussed in ACE Dev Channel, It has to do with the fact that I created a branch based off of 'Master' and then created a PR for this headbugfix branch into 'Release.'  I don't understand it. It looks like a GitHub bug to me.

**Comment by bux578** [comment]
"master" still has spectator whereas "release" doesn't/shouldn't.

**Comment by SilentSpike** [comment]
You can fix this with a rebase:
`git rebase --onto release master headbugfix`

It will require a `git push -f` though (to the headbugfix branch)

File: addons/optionsmenu/stringtable.xml (hunk 1 above)
Comment (ViperMaul) [inline addons/optionsmenu/stringtable.xml]: Good point.

Comment (gienkov) [inline addons/optionsmenu/stringtable.xml]: ```<Polish>Napraw headbug</Polish>```

File: addons/optionsmenu/stringtable.xml [hunk 2]
```diff
@@ -345,6 +345,12 @@
...
+        </Key>
+		<Key ID="STR_ACE_OptionsMenu_headBugFixTooltip">
+            <English>Resets your animation state.</English>
+        </Key>
```
Comment (gienkov) [inline addons/optionsmenu/stringtable.xml]: ```<Polish>Resetuje aktualną animację.</Polish>```

**Comment by gienkov** [comment]
I will be glad if you include my translations ;).

File: addons/common/functions/fnc_headBugFix.sqf [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ * @Descr: Fixes animation issues that may get you stuck
+ * @Author: rocko
+ *
+ * @Arguments:
+ * @Return: nil
+ * @PublicAPI: true
+ */
+
+#include "script_component.hpp"
+if (player != vehicle player) exitWith {};
```
Comment (Glowbal) [inline addons/common/functions/fnc_headBugFix.sqf]: Should use `ACE_player`.

Comment (Glowbal) [inline addons/common/functions/fnc_headBugFix.sqf]: Also it is lacking any kind of check for for example the unconscious state. It is possible to use fix headbug while unconscious, even though it's a valid state. I don't think that should be the case, even though most likely you will be moved back into the unconscious animation after.

**Comment by SilentSpike** [comment]
I'm thinking maybe we should just stage this for insertion into 3.3.0?

**Comment by ViperMaul** [comment]
This is a direct port from ACE2 as requested by the community. We can improve on it after 3.2.0

**Pull request merged**